### PR TITLE
[fix][doc] Fix links to broker configuration on Admin API Topics page

### PR DESCRIPTION
**Pull request by alpreu**
*(If this PR fixes a github issue, please add `Fixes #<xyz>`.)*

Fixes #<xyz>

*(or if this PR is one task of a github issue, please add `Master Issue: #<xyz>` to link to the master issue.)*

Master Issue: #<xyz>

### Motivation

The links to the broker configuration on the Admin API Topics page do not link to the desired page but to the general configuration page instead.

### Modifications

Change links to go directly to broker config page.

### Verifying this change

- [ ] Make sure that the change passes the CI checks.

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

### Does this pull request potentially affect one of the following parts:

*If `yes` was chosen, please highlight the changes*

  - Dependencies (does it add or upgrade a dependency): (yes / **no**)
  - The public API: (yes / **no**)
  - The schema: (yes / **no** / don't know)
  - The default values of configurations: (yes / **no**)
  - The wire protocol: (yes / **no**)
  - The rest endpoints: (yes / **no**)
  - The admin cli options: (yes / **no**)
  - Anything that affects deployment: (yes / **no** / don't know)

### Documentation

Check the box below or label this PR directly.

Need to update docs? 

- [ ] `doc-required` 
(Your PR needs to update docs and you will update later)
  
- [ ] `doc-not-needed` 
(Please explain why)
  
- [x] `doc` 
(Your PR contains doc changes)

- [ ] `doc-complete`
(Docs have been already added)

### CODE REVIEW

File: site2/docs/admin-api-topics.md [hunk 1]
```diff
@@ -1666,7 +1666,7 @@ Non-partitioned topics must be explicitly created. When creating a new non-parti
 
 By default, 60 seconds after creation, topics are considered inactive and deleted automatically to avoid generating trash data. To disable this feature, set `brokerDeleteInactiveTopicsEnabled` to `false`. To change the frequency of checking inactive topics, set `brokerDeleteInactiveTopicsFrequencySeconds` to a specific value.
 
-For more information about the two parameters, see [here](reference-configuration.md#broker).
+For more information about the two parameters, see [here](reference-configuration-broker.md).
```
Comment (Anonymitaet) [inline site2/docs/admin-api-topics.md:1669]: Hi @alpreu we're generating configuration docs from code automatically, and the new page will be something like https://pulsar.apache.org/tools/pulsar-config/2.11.0-SNAPSHOT/#/

`reference-configuration-broker.md` will be removed from the pulsar repo https://github.com/apache/pulsar/pull/17197

So can you hold on this change?

Feel free to update it after the new page is finalized, thank you!

/cc @SignorMercurio

Comment (Anonymitaet) [reply site2/docs/admin-api-topics.md:1669]: cc @momo-jun @daveduggins

Comment (SignorMercurio) [reply site2/docs/admin-api-topics.md:1669]: @Anonymitaet Note that the new link should be https://pulsar.apache.org/reference

Comment (Anonymitaet) [reply site2/docs/admin-api-topics.md:1669]: @SignorMercurio yes, thanks for reminding me.

Comment (alpreu) [reply site2/docs/admin-api-topics.md]: Hi together, I just went through the current site and updated most occurrences of the reference configuration.
There are a few links left that I could not find an equivalent for, maybe the documentation is outdated for these.
I'm attaching a screenshot below so you can see which ones are still missing:
Screenshot 2022-09-06 at 16 07 37

File: site2/docs/admin-api-brokers.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Pulsar brokers consist of two components:
 
 In addition to being configurable when you start them up, brokers can also be [dynamically configured](#dynamic-broker-configuration).
 
-For a full listing of broker-specific configuration parameters, see the [Configuration](reference-configuration.md#broker) page.
+For a full listing of broker-specific configuration parameters, see the [Configuration](https://pulsar.apache.org/reference/#/config/reference-configuration-broker) page.
```
Comment (Anonymitaet) [inline site2/docs/admin-api-brokers.md:38]: This URL can not be used since we're working in progress and it might be changed later, right? @SignorMercurio

**Review by Anonymitaet — CHANGES_REQUESTED** [review]

File: site2/docs/admin-api-topics.md (hunk 1 above)
Comment (Anonymitaet) [reply site2/docs/admin-api-topics.md]: Thanks! @alpreu 
I suggest holding these changes since this work is still WIP. @SignorMercurio thoughts?

Comment (SignorMercurio) [reply site2/docs/admin-api-topics.md]: Sorry, missed the notification. I agree that we should hold these changes.d

File: site2/docs/admin-api-brokers.md (hunk 2 above)
Comment (SignorMercurio) [reply site2/docs/admin-api-brokers.md:38]: Right.

**Review by tisonkun — CHANGES_REQUESTED** [review]
The source of docs pages is moved to pulsar-site repo. Please open the patch against the new location.

You can also edit the source file by clicking "Edit this page" button on the specific page.

**Comment by tisonkun** [comment]
Closing...

**Pull request closed without merging**